scenarios/evaluate/evaluate_custom/evaluate_custom.ipynb

```diff
@@ -233,7 +235,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "pf-test-record",
+   "display_name": "venv-azureai-samples",
```

Comment:
tip: nb-clean which is configured via pre-commit will remove all the metadata from your notebooks before you commit